main.go

```diff
@@ -268,18 +268,40 @@ func main() {
 	}
 
 	// Detect xcpretty version
-	if configs.OutputTool == "xcpretty" {
-		if !utils.IsXcprettyInstalled() {
-			fail(`xcpretty is not installed
-For xcpretty installation see: 'https://github.com/supermarin/xcpretty',
-or use 'xcodebuild' as 'output_tool'.`)
-		}
+	outputTool := configs.OutputTool
+	if outputTool == "xcpretty" {
+		fmt.Println()
+		log.Infof("Checking if output tool (xcpretty) is installed")
 
-		xcprettyVersion, err := utils.XcprettyVersion()
+		installed, err := utils.IsXcprettyInstalled()
 		if err != nil {
-			fail("Failed to determin xcpretty version, error: %s", err)
+			log.Warnf("Failed to check if xcpretty is installed, error: %s", err)
+			log.Printf("Switching to xcodebuild for output tool")
+			outputTool = "xcodebuild"
+		} else {
+			var err error
+			if !installed {
+				log.Warnf(`xcpretty is not installed`)
+				fmt.Println()
+				log.Printf("Installing xcpretty")
+
+				if err = utils.InstallXcpretty(); err != nil {
+					log.Warnf("Failed to install xcpretty, error: %s", err)
+					log.Printf("Switching to xcodebuild for output tool")
+					outputTool = "xcodebuild"
+				}
+			}
+
+			if err == nil {
+				xcprettyVersion, err := utils.XcprettyVersion()
+				if err != nil {
+					log.Warnf("Failed to determin xcpretty version, error: %s", err)
+					log.Printf("Switching to xcodebuild for output tool")
+					outputTool = "xcodebuild"
+				}
+				log.Printf("- xcprettyVersion: %s", xcprettyVersion.String())
```

Comment:
do this after you made sure that xcpretty is available or switched to xcodebuild (if outputtool == xcpretty {...)